app/controllers/nsm/steps/view_claim_controller.rb

```diff
@@ -18,53 +18,89 @@ def item
         render report_params[:item_type]
       end
 
-      def work_items
+      def claimed_work_items
+        @report = CheckAnswers::ReadOnlyReport.new(current_application)
```

Comment:
Setting @report and @claim in every action doesn't feel very DRY. We might be able to do away with `@claim` entirely, cos I think you might be able to reference `current_application` directly from the view. But failing that, maybe we could set these in a `before_action`?

Reply:
Was just matching the existing convention but yeah I agree